nixpkgs/nixpkgs.bzl

```diff
@@ -36,12 +35,30 @@ def _nixpkgs_local_repository_impl(repository_ctx):
             content = repository_ctx.attr.nix_file_content,
             executable = False,
         )
-        target = repository_ctx.path("default.nix")
+        target_filename = "default.nix"
+        target = repository_ctx.path(target_filename)
     else:
-        target = _cp(repository_ctx, repository_ctx.attr.nix_file)
+        target_filename = repository_ctx.attr.nix_file
+        target = _cp(repository_ctx, target_filename)
 
+    print(repository_ctx.attr.nix_file_deps)
+    dep_files = []
     for dep in repository_ctx.attr.nix_file_deps:
-        _cp(repository_ctx, dep)
+        dest = _cp(repository_ctx, dep)
+        dep_files.append(dest)
+
+    # Export all specified Nix files to make them dependencies of a
+    # nixpkgs_package rule.
+    export_files = 'exports_files(["{}", {}])'.format(
+        target_filename.name,
+        ",".join(['"{}"'.format(p) for p in dep_files]))
```

Comment:
A shorter and arguably more readable way of writing this would be,

```python
export_files = "exports_files({})".format(str([target_filename.name] + dep_files))
```

Reply:
Ahah, I agree that's really more readable! Fixed.